app/models/refusal_advice.rb

```diff
@@ -0,0 +1,27 @@
+# A collection of Questions that help users challenge refusals.
+class RefusalAdvice
+  def self.default
+    files = Rails.configuration.paths['config/refusal_advice'].existent
+    new(RefusalAdvice::Data.from_yaml(files))
```

Comment:
Zarino is getting an error along the lines of "Undefined method `from_yaml` on class `Data`". His VM is using Ruby 2.3. Works for me on 2.5 and doesn't fail in CI.